src/openms/source/ANALYSIS/ID/IDScoreGetterSetter.cpp

```diff
@@ -95,6 +154,8 @@ namespace OpenMS
       }
       // if for none of the proteins the target version was picked, add as decoy
       if (decoy_picked) scores_labels.emplace_back(grp.probability, 0.0);
+      // TODO I think we need an unordered_set to check which proteins were picked already
```

Comment:
so this algo works in its current state or is this something that needs to be implemented?

Reply:
Still needs to be implemented. In this way it still can count pairs multiple times

Reply:
I think it is fine for now. I think it is normal that there will be slight differences on FDR due to grouping.

Reply:
But the warning comment can stay in case someone wonders why things are counted that way.